tests/acceptance/00_basics/04_bundles/dynamic_bundlesequence/dynamic_inputs_based_on_list_variable_dependent_on_class.cf

```diff
@@ -0,0 +1,25 @@
+body common control
+{
+    inputs         => { @(inventory.inputs)  };
+    bundlesequence => { @(inventory.bundles) };
```

Comment:
I can't comment on our actual behavior, CFEngine is so subtle in this area, but it looks "intuitively" correct to me.